g2p_seq2seq/seq2seq_model.py

```diff
@@ -333,29 +335,33 @@ def get_eval_set_batch(self, data, bucket_id, from_row_id):
       batch_row_id += 1
 
     return self.__create_batch_major_vecs(encoder_size, decoder_size,
-                                          encoder_inputs, decoder_inputs)
+                                          encoder_inputs, decoder_inputs,
+                                          batch_row_id)
 
 
   def __create_batch_major_vecs(self, encoder_size, decoder_size,
-                                encoder_inputs, decoder_inputs):
+                                encoder_inputs, decoder_inputs,
+                                batch_size_actual=None):
```

Comment:
You do not need this parameter. You can simply use len(encoder_inputs) and len(decoder_inputs).

It is also not quite clear why do you create so many lists and arrays, you could fill required array much more efficiently without creating lists.